pre_commit/commands/run.py

```diff
@@ -190,7 +190,7 @@ def _compute_cols(hooks, verbose):
 def _all_filenames(args):
     if args.origin and args.source:
         return git.get_changed_files(args.origin, args.source)
-    elif args.hook_stage == 'commit-msg':
+    elif args.hook_stage in ['prepare-commit-msg', 'commit-msg']:
```

Comment:
very minor, this should be a set or tuple literal instead of a list literal `in {'prepare-commit-msg', 'commit-msg'}`